scripts/helmcharts/vars.yaml

```diff
@@ -60,6 +67,8 @@ minio:
       # `openssl rand -hex 20`
       accessKey: "changeMeMinioAccessKey"
       secretKey: "changeMeMinioPassword"
+      minioHost: minio.db.svc.cluster.local
+      minioPort: 9000
```

Comment:
You don't have to add a new host here, as the endpoint value is already available [here](https://github.com/openrepleay/openreplay/blob/main/scripts/helmcharts/vars.yaml#L107)

Use a method like [this](https://github.com/openreplay/openreplay/blob/main/scripts/helmcharts/openreplay/charts/sink/templates/deployment.yaml#L68) to get the necessary value fields.

Reply:
same as above, resolved with https://github.com/openreplay/openreplay/pull/757/commits/dd147fae9bdc428e542f9d85e9b8681c330b5c75